dbt_project.yml

```diff
@@ -1,11 +1,11 @@
 name: 'dbt_utils'
-version: '0.7.0'
+version: '0.1.0'
 
-require-dbt-version: [">=0.20.0", "<=1.0.0"]
+require-dbt-version: [">=1.0.0-rc", "<2.0.0"]
```

Comment:
What do you think about switching this to `[">=1.0.0", "<2.0.0"]` once we have a final release of dbt-core v1.0.0? In which case, this feels fine for now

Reply:
Yeah I don't really want to have to release 0.8.1 in a week that just drops the release candidates. Since
- 0.7.5 supports <= 1.0 as its upper bound, which _does_ include the RCs, and
- 1.0 final is on the cusp of release anyway,

 I think I'm happy to say that 0.8 is only going to cover the final released version of 1.0

Reply:
With that said, that means that when I merge this in, Circle CI will give it a ❌ until 1.0 final actually ships. Do we feel OK about that? (cc @jasnonaz)